client/client.go

```diff
@@ -188,6 +214,284 @@ func (q Query) AddResponse(resp *http.Response) {
 	}
 }
 
+// Request processing
+
+// ExecuteWebsocketQuery handles Websocket queries
+func ExecuteWebsocketQuery(query Query) {
+	url := query.URL()
+	c, resp, err := websocket.DefaultDialer.Dial(url, query.Headers())
+	if query.CheckErr(err) {
+		return
+	}
+	defer c.Close()
+	query.AddResponse(resp)
+	messages := query["messages"].([]interface{})
+	for _, msg := range messages {
+		err = c.WriteMessage(websocket.TextMessage, []byte(msg.(string)))
+		if query.CheckErr(err) {
+			return
```

Comment:
Log the error before returning?

Reply:
I didn't change this code either. But my general comment from above applies here as well. I think logging makes sense if it will help someone debug this code. Otherwise it is noise.